.semaphore/prologue.sh

```diff
@@ -41,3 +41,50 @@ sudo rm -rf \
     /opt/firefox-esr-prev \
     /usr/local/golang \
     /mnt/docker.qcow2
+
+# Originally found on channable/requestmachine/.semaphore/prologue.sh
```

Comment:
Whoa!  Thanks for spotting this.

I removed the reference -- and squashed my commits into a new one that does not contain the above reference so that it does not show up in history after merging into master.

Reply:
I will try to delete this thread after the PR is closed so the reference does not show up here.